### PR TITLE
doc: add minutes for meeting 23 Jan 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-01-23.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    but Gabriel ‘volunteered’ to confirm process and comment in 651 on what specifically to
+    do.
+  * Gabriel will also create a new issue to continue discussion around whether we want
+    to assign bits etc. (PoC is in https://github.com/nodejs/node/pull/25539)
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles today so no update.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Steams, stalled waiting on people who know more about current streams implementation
```
Comment (billouboq) [inline meetings/2019-01-23.md]: * streams ?

Comment (mhdawson) [reply meetings/2019-01-23.md]: @billouboq thanks, fixed.

File: meetings/2019-01-23.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  * Gabriel will also create a new issue to continue discussion around whether we want
+    to assign bits etc. (PoC is in https://github.com/nodejs/node/pull/25539)
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles today so no update.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Streams, stalled waiting on people who know more about current streams implementation
+  * N-API
+    * Focus on getting tfsn non-experimental and backported in time for planned 8.x SemVer minor
```
Comment (Trott) [inline meetings/2019-01-23.md]: tfsn? Is that a typo or an acronym? If an acronym, maybe spell it out and/or link to something that explains it?

File: meetings/2019-01-23.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    but Gabriel ‘volunteered’ to confirm process and comment in 651 on what specifically to
+    do.
+  * Gabriel will also create a new issue to continue discussion around whether we want
+    to assign bits etc. (PoC is in https://github.com/nodejs/node/pull/25539)
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles today so no update.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Streams, stalled waiting on people who know more about current streams implementation
```
Comment (Trott) [inline meetings/2019-01-23.md]:
```suggestion
  * Streams: Stalled. Waiting on people who know more about current streams implementation.
```

File: meetings/2019-01-23.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2019-01-23.md]: expanded

**Pull request merged**